manifests/scripts/metrics-dashboard/run.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
Is this script and the associated yaml files only used to enable us to create our grafana dashboard (i.e. for [GITOPSRVCE-340](https://issues.redhat.com/browse/GITOPSRVCE-340))?

Reply:
@jopit correct. It's primarily for us to run locally on our own machines, in order to work on new metrics or Grafana dashboards. (Thus it doesn't need to be production ready, etc. Just for our use as devs)

In the Stone Soup staging and production environments, they have they own mechanism for setting up Prometheus and Grafana: https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring